rdb/src/main/java/org/openmbee/sdvc/rdb/config/DatabaseDefinitionService.java

```diff
@@ -34,11 +34,11 @@
 public class DatabaseDefinitionService {
 
     private static final Pattern pattern = Pattern.compile("\\$\\{(.+?)\\}");
-    private static final String COPY_SQL = "INSERT INTO %s SELECT * FROM %s";
-    private static final String COPY_IDX = "SELECT SETVAL('%s_id_seq', COALESCE((SELECT MAX(id) FROM %s), 1), true)";
+    private static final String COPY_SQL = "INSERT INTO \"%s\" SELECT * FROM \"%s\"";
+    private static final String COPY_IDX = "SELECT SETVAL('%s_id_seq', COALESCE((SELECT MAX(\"id\") FROM \"%s\"), 1), true)";
 
-    private static final String INITIAL_PROJECT = "INSERT INTO nodes (id, nodeid, docid, initialcommit, lastcommit, nodetype, deleted) VALUES (0, ?, ?, ?, ?, ?, false)";
-    private static final String INITIAL_REF = "INSERT INTO branches (id, branchid, branchname, tag, deleted, timestamp) VALUES (0, 'master', 'master', false, false, NOW());";
+    private static final String INITIAL_PROJECT = "INSERT INTO \"nodes\" (\"id\", \"nodeId\", \"docId\", \"initialCommit\", \"lastCommit\", \"nodeType\", \"deleted\") VALUES (0, ?, ?, ?, ?, ?, false)";
```

Comment:
don't think we need quotes here since this is for the master node table, same for the master branch below

Reply:
We might not need it.  I was just trying to make things look consistent.